[BUG] Swagger UI no longer loads

### Describe the bug

After updating from `7.67.3` to `7.68.2`. my swagger UI no longer loads. it seems the assets it tries to load just now 404

```
[2024-04-27T16:55:07.701] [ERROR] [TSED] - {
  method: 'GET',
  url: '/api-docs/swagger-ui-bundle.js',
  route: '/api-docs',
  headers: {
    host: '127.0.0.1:8081',
    connection: 'keep-alive',
    'sec-ch-ua': '"Chromium";v="124", "Google Chrome";v="124", "Not-A.Brand";v="99"',
    'sec-ch-ua-mobile': '?0',
    'sec-ch-ua-platform': '"Windows"',
    dnt: '1',
    'upgrade-insecure-requests': '1',
    'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36',
    accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.7',
    'sec-fetch-site': 'same-origin',
    'sec-fetch-mode': 'navigate',
    'sec-fetch-user': '?1',
    'sec-fetch-dest': 'document',
    'accept-encoding': 'gzip, deflate, br, zstd',
    'accept-language': 'en-GB,en;q=0.9',
    cookie: 'login_view=VictoriqueMoe; login=victoriquemoe; ALPHA_KEY=2726042b42a211d84c4a302c81510ee0c0170a9e8f2a87f6931be317cfee1cc9'
  },
  body: {},
  query: {},
  params: {},
  reqId: '3b80203505814fa0a30051b3c00e7234',
  time: 2024-04-27T15:55:07.701Z,
  duration: 3,
  event: 'request.end',
  status: 404,
  status_code: '404',
  state: 'KO'
}
```

In fact, no assets load, this includes `/swagger-ui-bundle.js`, `/swagger-ui-standalone-preset.js`, `/swagger-ui.css`, etc...

my swagger config has not changed: 

```ts
swagger: [
        {
            path: "/api-docs",
            specVersion: "3.0.3",
            options: {
                withCredentials: true,
                supportedSubmitMethods: isProduction ? [] : undefined,
            },
        },
    ],
```

I also tried the new `3.1.0` but that didn't do anything.

Did the config change and i am not aware?

### To Reproduce

1. set up swagger
2. try to load page

### Expected behavior

swagger UI loads

### Code snippets

_No response_

### Repository URL example

_No response_

### OS

Windows 11

### Node version

v20.11.0

### Library version

7.68.2

### Additional context

I am using ESM

--- a/packages/specs/swagger/scripts/constants.js
+++ b/packages/specs/swagger/scripts/constants.js
@@ -2,6 +2,6 @@ import {dirname} from "node:path";
 import {fileURLToPath} from "node:url";
 
 // @ts-ignore
-export const SWAGGER_UI_DIST = dirname(import.meta.resolve("swagger-ui-dist"));
+export const SWAGGER_UI_DIST = dirname(import.meta.resolve("swagger-ui-dist")).replace("file://", "");
 // @ts-expect-error
 export const ROOT_DIR = dirname(fileURLToPath(import.meta.url));

